geo-location.html

```diff
@@ -129,32 +139,47 @@
        */
 
       ready: function() {
+        this.fetch();
+      },
+
+      clear: function () {
+        this._setPosition(undefined);
```

Comment:
because that's what they are in the default scenario, before `fetch` has happened.